db/db_impl/db_secondary_test.cc

```diff
@@ -469,6 +469,7 @@ TEST_F(DBSecondaryTest, MissingTableFile) {
         if (s.IsPathNotFound()) {
           ++table_files_not_exist;
         } else if (!s.ok()) {
+          printf("MJR: Status=%s\b", s.ToString().c_str());
```

Comment:
what does MJR stand for?

Reply:
Sorry.  my initials.  Leftover debugging.  I will remove it.